<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Jagadish" failures="1" tests="8" name="runners.Test_DemoBlaze_e2e_runner" time="58.010" errors="0" timestamp="2024-12-29T19:43:28 IST" skipped="0">
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="21.790"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="8.176"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="5.049"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="4.260"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="5.294">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepdefinitions.Signup.user_should_successfully_signup_and_should_be_display(Signup.java:52)
at âœ½.User should successfully signup and "Sign up successful." should be display(file:///C:/Users/Jagadish%20Moharana/OneDrive/Desktop/masai%20courses/cucumber/DemoBlaze_Cucumber_project/src/test/resources/features/Signup.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="3.474"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="5.028"/>
  <testcase classname="runners.Test_DemoBlaze_e2e_runner" name="runScenario" time="4.939"/>
</testsuite> <!-- runners.Test_DemoBlaze_e2e_runner -->
